src/Runner.Worker/Expressions/HashFilesFunction.cs

```diff
@@ -33,25 +33,41 @@ public sealed class HashFilesFunction : Function
             string githubWorkspace = workspaceData.Value;
             bool followSymlink = false;
             List<string> patterns = new List<string>();
-            var firstParameter = true;
             foreach (var parameter in Parameters)
             {
                 var parameterString = parameter.Evaluate(context).ConvertToString();
-                if (firstParameter)
+                if (parameterString.StartsWith("--"))
                 {
-                    firstParameter = false;
-                    if (parameterString.StartsWith("--"))
+                    if (string.Equals(parameterString, "--follow-symbolic-links", StringComparison.OrdinalIgnoreCase))
                     {
-                        if (string.Equals(parameterString, "--follow-symbolic-links", StringComparison.OrdinalIgnoreCase))
+                        followSymlink = true;
+                        continue;
+                    }
+                    else if (parameterString.StartsWith("--timeout=", StringComparison.OrdinalIgnoreCase))
+                    {
+                        var timeoutSecondsString = parameterString.Split("=")[1];
+                        if (!String.IsNullOrEmpty(timeoutSecondsString))
                         {
-                            followSymlink = true;
-                            continue;
+                            bool parseSuccess = int.TryParse(timeoutSecondsString, out int timoutSeconds);
```

Comment:
Please check the typo, `timout` → `timeout`
If we're going to get all nitpicky about it, `parseSuccess` → `parsingSuccess`

Reply:
Fixed, thanks!